### PR TITLE
Implement Projects page, similar to "Orders" page.

### PR DESCRIPTION
**Pull request by Yurashk**

### CODE REVIEW

File: src/app/src/_data/repository/order/model/order.response.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import {PRODUCT_ATTRIBUTE_NAMES} from '../../product/model';
```
Comment (ultramarine256) [inline src/app/src/_data/repository/order/model/order.response.ts]: Redundant PRODUCT_ATTRIBUTE_NAMES ?

File: src/app/src/_data/repository/project/project.repository.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  /// methods
+  // public placeOrder(woocommerceOrder: {}): Observable<OrderResponse> {
+  //   return this.httpClient
+  //     .post<OrderCreateModel>(`${this.apiBaseUrl}/wp-json/onshop/v1/order`, woocommerceOrder)
+  //     .pipe(map(dto => {
+  //       const model = new OrderResponse();
+  //       model.mapFromDto(dto);
+  //       return model;
+  //     }));
+  // }
```
Comment (ultramarine256) [inline src/app/src/_data/repository/project/project.repository.ts]: Removed unused code.

File: src/app/src/_domain/modules/profile/orders/orders.component.scss [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+}
 
+@media only screen and (max-width: 600px){
+  .font-less{
+    font-size: 0.8em;
+  }
+  .hide-element{
+    display: none;
+  }
+}
```
Comment (ultramarine256) [inline src/app/src/_domain/modules/profile/orders/orders.component.scss]: Combine styles in to nested tree.

File: src/app/src/_presentation/pages/profile/projects/projects-page.component.ts [hunk 4]
```diff
@@ -1,11 +1,27 @@
 import {Component} from '@angular/core';
 import {AuthService} from '../../../../_domain';
+import {ProjectRepository} from '../../../../_data/repository/project/project.repository';
+import {ProjectResponse} from '../../../../_data/repository/project/model';
```
Comment (ultramarine256) [inline src/app/src/_presentation/pages/profile/projects/projects-page.component.ts]: Simplify import path.

File: src/app/src/_data/repository/project/project.repository.ts (hunk 2 above)
Comment (Yurashk) [reply src/app/src/_data/repository/project/project.repository.ts]: Done.

**Pull request merged**